lib/protocol/networks.js

```diff
@@ -899,23 +897,23 @@ simnet.pow.targetTimespan =
   simnet.pow.targetWindow * simnet.pow.targetSpacing;
 simnet.pow.minActual = (simnet.pow.targetTimespan / 4) >>> 0;
 simnet.pow.maxActual = simnet.pow.targetTimespan * 4;
-simnet.pow.targetReset = false;
+simnet.pow.targetReset = true;
 simnet.pow.noRetargeting = false;
 
 simnet.names = {
   auctionStart: 0,
   rolloutInterval: 1,
   lockupPeriod: 1,
-  renewalWindow: 2500,
+  renewalWindow: simnet.pow.blocksPerDay,
   renewalPeriod: 1250,
-  renewalMaturity: 25,
+  renewalMaturity: 1,
   claimPeriod: 75000,
-  biddingPeriod: 25,
-  revealPeriod: 50,
+  biddingPeriod: (simnet.pow.blocksPerDay / 2) | 0,
+  revealPeriod: (simnet.pow.blocksPerDay / 2) | 0,
```

Comment:
Thoughts about making this a day long rather than 12 hours? Not that the names on this network matter all that much, but it may make for some awkward times needed to reveal a name (e.g. Bid at 10am, renewal is overnight from 10PM to 10am).